shortcodes.php

```diff
@@ -607,23 +624,23 @@ function sc_ucf_video($atts = Array())
  * @return string
  * @author Chris Conover
  **/
-function sc_resources($atts = Array())
+function sc_resources( $atts = array() )
 {
-	$css = (isset($atts['css'])) ? $atts['css'] : '';
+	$css = ( isset( $atts['css'] ) ) ? $atts['css'] : '';
 
-	$args = Array('menu' => 'Resources', 'container' => '', 'menu_id'=> '');
+	$args = array( 'menu' => 'Resources', 'container' => '', 'menu_id'=> '' );
 
 	ob_start();
 	?>
-	<div class="<?=$css?>" id="resources">
+	<div class="<?php echo $css; ?>" id="resources">
 		<h2>Resources</h2>
-		<?=wp_nav_menu($args)?>
+		<?php echo wp_nav_menu( $args ); ?>
```

Comment:
`echo` isn't necessary when calling `wp_nav_menu()`; this function echo's its results by default